planner/core/expression_rewriter.go

```diff
@@ -318,7 +318,7 @@ func (er *expressionRewriter) constructBinaryOpFunction(l expression.Expression,
 
 // buildSubquery translates the subquery ast to plan.
 // Currently, only the EXIST can apply the rewrite hint(rewrite the semi join to inner join with aggregation).
-func (er *expressionRewriter) buildSubquery(ctx context.Context, subq *ast.SubqueryExpr, rewriteHintCanTakeEffect bool) (np LogicalPlan, hasSemiJoinRewriteHint bool, err error) {
```

Comment:
Please update the comments for this function.

Reply:
Updated.